src/LocalElection/CandidateImportCommand.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\LocalElection;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class CandidateImportCommand
+{
+    /**
+     * @Assert\File(
+     *     maxSize="5M",
+     *     mimeTypes={
+     *         "text/plain",
+     *         "text/csv",
+     *     }
+     * )
+     */
+    private ?UploadedFile $file;
```

Comment:
```suggestion
    public ?UploadedFile $file = null;
```
and you can remove get/set methods